playwright.config.ts

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 import { PlaywrightTestConfig } from '@playwright/test';
-import path from 'path';
 
 const config: PlaywrightTestConfig = {
   timeout: 120 * 1000,
   testDir: './tests',
+  reporter: [['list'], ['github']],
```

Comment:
drive-by: dot reporter does us not give us fast feedback inside a CI run since it splits only after 80 dots.
drive-by: dog GitHub reporter.